src/main/java/com/hillel/items_exchange/dto/UserDto.java

```diff
@@ -0,0 +1,40 @@
+package com.hillel.items_exchange.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.*;
+import java.time.LocalDate;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class UserDto {
+    private Long id;
+    @NotEmpty(message = "Username has to be not empty")
+    @Size(min = 3, max = 50, message = "Username has to be between 3 and 50 symbols")
```

Comment:
Please change it into @Size(min = 2, max = 50, message = "Your login must be between 2 and 50 characters")